### PR TITLE
New post: Ingesting Data into OpenSearch using Apache Kafka and Go

### PR DESCRIPTION
**Pull request by abhirockzz**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by cobusbernard** [comment]
Just waiting on the code sample PR.

**Comment by abhirockzz** [comment]
> Just waiting on the code sample PR.

Done https://github.com/build-on-aws/opensearch-using-kafka-golang/pull/1

File: tutorials/opensearch-kafka-golang-data-ingestion/index.md [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+## Clean up
+
+After you are done with the demo, make sure to delete all the services to avoid incurring any additional charges. You can follow the steps in the respective documentation to delete the services.
+
+- [Delete OpenSearch Serverless collection](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-manage.html#serverless-delete?sc_channel=el&sc_campaign=datamlwave&sc_content=opensearch-kafka-golang&sc_geo=mult&sc_country=mult&sc_outcome=acq)
+- [Delete MSK Serverless cluster](https://docs.aws.amazon.com/msk/latest/developerguide/delete-resources.html?sc_channel=el&sc_campaign=datamlwave&sc_content=opensearch-kafka-golang&sc_geo=mult&sc_country=mult&sc_outcome=acq)
+- [Delete Cloud9 environment](https://docs.aws.amazon.com/cloud9/latest/user-guide/delete-environment.html?sc_channel=el&sc_campaign=datamlwave&sc_content=opensearch-kafka-golang&sc_geo=mult&sc_country=mult&sc_outcome=acq)
+- Also delete IAM roles and policies
+
+## Conclusion
```
Comment (riferrei) [inline tutorials/opensearch-kafka-golang-data-ingestion/index.md:416]: Maybe before wrapping up the blog post you could list the article below as one of the next steps, if the readers wants to write data into OpenSearch directly from Go.

File: tutorials/opensearch-kafka-golang-data-ingestion/index.md [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+
+Sometimes though, there are specific requirements in your data pipeline which might need you to write your own integration layer. This is not a problem since both Kafka and OpenSearch provide client libraries for a variety of programming languages. This is exactly what's covered in this blog! We will make use of a custom [Go](http://go.dev/) application to ingest data using Go clients for [Kafka](https://github.com/twmb/franz-go) and [OpenSearch](https://opensearch.org/docs/latest/clients/go/).
```
Comment (riferrei) [inline tutorials/opensearch-kafka-golang-data-ingestion/index.md:34]: I think you should spend more time in this section to describe the purpose of creating a custom integration layer in Go. By looking at the tutorial, the app is a producer and consumer, which raises the question: why not use Kafka Connect instead?

I know what you mean by custom scenarios and the need to write a custom integration layer, but in this case is not very clear to the reader.

**Review by riferrei — APPROVED** [review]
Couple of comments added, but LGTM in general

**Review by cobusbernard — APPROVED** [review]

**Pull request merged**